Platform.PCL/Realm.Sync.PCL/UserPCL.cs

```diff
@@ -28,6 +28,46 @@ namespace Realms.Sync
     /// </summary>
     public class User : IEquatable<User>
     {
+        #region static
+
+        /// <summary>
+        /// Gets the currently logged-in user. If none exists, null is returned. If more than one user is currently logged in, an exception is thrown.
```

Comment:
What does _logged-in_ mean? This seems to make assumptions about only having a single server connection.